apps/video/test/marionette/manifest.ini

```diff
@@ -1,2 +1,3 @@
 [video_list_test.js]
 [overlay_test.js]
+[play_multiple_types_test.js]
```

Comment:
This file is actually unused and was just removed by bug 1225552 a couple of days ago. That's presumably why the patch is bitrotted.